src/content/.template.config/template.json

```diff
@@ -54,13 +63,32 @@
         {
           "condition": "(!IncludeTests)",
           "exclude": [ "**/AppNamePlaceholder.Tests/**/*" ]
+        },
+        {
+          "condition": "(!UsePaket)",
+          "exclude": [ "**/paket.*" ]
         }
       ]
     },
     {
       "source": "./DotLiquid/",
       "target": "./",
       "condition": "(ViewEngine == \"dotliquid\")",
+      "modifiers": [
+        {
+          "condition": "(!IncludeTests)",
+          "exclude": [ "**/AppNamePlaceholder.Tests/**/*" ]
+        },
+        {
+          "condition": "(!UsePaket)",
+          "exclude": [ "**/paket.*" ]
+        }
+      ]
+    },
+    {
+      "source": "./Paket/",
+      "target": "./",
+      "condition": "(UsePaket == true)",
```

Comment:
Not sure, but I think this could have even been reduced to `"condition": "(UsePaket)"`

Reply:
You are likely correct. I will try it out and amend.